opendevin/runtime/plugins/agent_skills/setup.sh

```diff
@@ -2,6 +2,7 @@
 
 set -e
 
+source ~/.bashrc
```

Comment:
Maybe you can move it into end of the file before the `pip install` and then we can remove the `export PATH` and `export PYTHONPATH`

Reply:
Or we can add a `$OD_PYTHON` env var, and change all the plugins to use it? (e.g., `$OD_PYTHON -m pip install XX`)

Reply:
I moved it before pip because sourcing ~/.bashrc is necessary to use Miniforge's pip; otherwise, it will use /usr/bin/pip. I'm not sure if there's a good way to resolve this.

Reply:
I mean you can change code from:
```shell
source ~/.bashrc
echo 'export PATH=/opendevin/plugins/agent_skills:$PATH' >> ~/.bashrc
export PATH=/opendevin/plugins/agent_skills:$PATH
echo 'export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH' >> ~/.bashrc
export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH
pip install flake8 python-docx PyPDF2 python-pptx pylatexenc openai opencv-python
```

to this

```shell
echo 'export PATH=/opendevin/plugins/agent_skills:$PATH' >> ~/.bashrc
echo 'export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH' >> ~/.bashrc
source ~/.bashrc
pip install flake8 python-docx PyPDF2 python-pptx pylatexenc openai opencv-python
```

Will this result pip didn't use Miniforge's pip?

Reply:
Thank you! It's done and works fine. Please take a look!